endpoints/cookie_sync_test.go

```diff
@@ -285,6 +298,43 @@ func TestCookieSyncHandle(t *testing.T) {
 				a.On("LogCookieSyncObject", &expected).Once()
 			},
 		},
+		{
+			description:                  "CookieDeprecation not set in repsonse", // CookieDeprecation enabled for account, receive-cookie-deprecation cookie present in request, cookie deprecation header not set in response
```

Comment:
I think a better test here would be to exercise that the cookie is set in the response rather than it already being set.
All you need to change to make this happen is to:
- modify the description
- delete all references to `givenChromeDeprecationCookie` including the conditional with `request.AddCookie(&http.Cookie{Name: receiveCookieDeprecation, Value: "1"})`
- set `expectedCookieDeprecationHeader` to true in this test case

You can then lean on your unit test `TestSetCookieDeprecationHeader` for the rest of your coverage.

Reply:
done